source/common/config/grpc_stream.h

```diff
@@ -128,7 +128,8 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
   ControlPlaneStats generateControlPlaneStats(Stats::Scope& scope) {
```

Comment:
Can we remove this method and use the new utility method you have added?

Reply:
Done.